pkg/solidity-utils/test/foundry/WeightedMathRounding.t.sol

```diff
@@ -215,7 +215,7 @@ contract WeightedMathRoundingTest is Test {
         bool flipBit
     ) external view {
         uint256[] memory weights = new uint256[](2);
-        uint256[] memory balances = new uint256[](2);
+        uint256[] memory balances = new uint256[](rawBalances.length);
```

Comment:
Might as well leave it as 2, since it's hard-coded in the arguments anyway. Then there'd be no diff for this file.